README.md

```diff
@@ -219,6 +219,31 @@ const usernameInputElement = getByTestId('username-input')
 > Learn more about `data-testid`s from the blog post
 > ["Making your UI tests resilient to change"][data-testid-blog-post]
 
+#### `toBeInTheDOM`
```

Comment:
Could we put these in a different section of the README (maybe right where it is).

So just change this to:

```md
## Custom Jest Matchers

There are two simple custom Jest matchers which you can use to extend the `expect` API of Jest for making assertions easier.

### `toBeInTheDOM`

This allows you to assert whether or not an element present in the DOM

... etc.
```